app/controllers/registrations_controller.rb

```diff
@@ -8,23 +8,22 @@ def edit
     render "edit"
   end
 
-# we need this subclassed method so that Devise doesn't force people to
-# change their password every time they want to edit their settings.
-# we also check that they give their current password to change their password.
-# Code copied from
-# https://github.com/plataformatec/devise/wiki/How-To:-Allow-users-to-edit-their-account-without-providing-a-password
+  # we need this subclassed method so that Devise doesn't force people to
+  # change their password every time they want to edit their settings.
+  # we also check that they give their current password to change their password.
+  # Code copied from
+  # https://github.com/plataformatec/devise/wiki/How-To:-Allow-users-to-edit-their-account-without-providing-a-password
 
   def update
-
     @member = Member.find(current_member.id)
 
-    successfully_updated = if needs_password?(@member, params)
-      @member.update_with_password(devise_parameter_sanitizer.sanitize(:account_update))
+    if needs_password?(@member, params)
+      successfully_updated = @member.update_with_password(devise_parameter_sanitizer.sanitize(:account_update))
```

Comment:
It does, but here's an ignore for that file.

another PR that fixes yet more coming soon.